helix-front/server/controllers/helix.ts

```diff
@@ -41,6 +42,12 @@ export class HelixCtrl {
 
     if (apiPrefix) {
       const realUrl = apiPrefix + url.replace(`/${helixKey}`, '');
+      console.log('realUrl from helix api proxy');
+      console.log(realUrl);
+      console.log('');
+      console.log('request body');
+      console.log(req.body);
```

Comment:
I'd like to leave this logging in the server logs, to aid debugging similar issues in the future. I'm happy to modify the formatting if reviewers recommend this.